cmd/derper/derper.go

```diff
@@ -76,6 +76,8 @@ var (
 	tcpKeepAlive = flag.Duration("tcp-keepalive-time", 10*time.Minute, "TCP keepalive time")
 	// tcpUserTimeout is intentionally short, so that hung connections are cleaned up promptly. DERPs should be nearby users.
 	tcpUserTimeout = flag.Duration("tcp-user-timeout", 15*time.Second, "TCP user timeout")
+
+	showTailscaleSpecificAbuseInformation = flag.Bool("show-tailscale-abuse-info", false, "whether to show Tailscale-specific abuse information on home page.")
```

Comment:
Let's remove this flag and do it as a function of the `--hostname` flag value instead. We do that elsewhere already.

Then we don't have to change service configuration either.

Reply:
Done